install-dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+cd $(dirname $0)
+pip install -e $(ls ./*/pyproject.toml | sed 's/\/pyproject.toml/\[dev\]/')
```

Comment:
two notes:
1. the `-e` flag has to be passed before each package you want to be editable 
2. i think it would be better to just hard-code all the client names to avoid any ambiguity, e.g.:
```suggestion
pip install -e ./checks-superstaq[dev] -e ./general-superstaq[dev] -e ./cirq-superstaq[dev] -e ./qiskit-superstaq[dev] -e ./superstaq-benchmarks[dev]
```

Reply:
This gives me an error:
```zsh
$ ./dev_tools/install.sh
cd ./dev_tools/../
ERROR: ./superstaq-benchmarks[dev] is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
```

Reply:
I think it should go away with `./superstaq-benchmarks[dev]` ->  `./supermarq-benchmarks[dev]`

Reply:
Oh right :sweat_smile:

Reply:
lol my bad